lib/middleware/websocket/interactors/payloads/player_join_payload.rb

```diff
@@ -0,0 +1,9 @@
+module Websocket
+  module Interactor
+    class PlayerJoinPayload
+      def call(player_id:, player_name:)
+        { 'id' => player_id, 'name' => player_name }.to_json
```

Comment:
We could consider making this an actual entity and `.to_json` that, instead of just a Hash. That way if `player_id` or `player_name` aren't types that we expect (they are passed as Dates for example) our code will explode instead of continuing along in a possibly incorrect state.